static/src/js/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+/**
+ * Renders the `App` component
+ *
+ * @component
+ * @example <caption>Renders a `App` component</caption>
+ * return (
+ *   <App />
+ * )
+ */
+export const App = () => (
+  <div>Hello World!</div>
+)
```

Comment:
Hello World, not something like "loading"? I assume this get's replaced by an app, but what happens if the app fails to load?

Reply:
I removed this